### ISSUE TITLE
PHP Error

### ISSUE THREAD

**Issue by bhsmither** [cubecart/v6#595]
When installing (not upgrading), index.php builds the array `$default_config_settings`, of which `$cookie_domain` is part.

Later, setup.install.php builds the array `$config_settings` and wants to store that array in `Config->set('config')`. However, `$glob` is also passed to Config and in `Config->__construct()`, line77, PHP complains because `$this->_config['config']['cookie_domain']` does not yet exist because it has yet to be set/databased.

This is a trivial error, but maybe `$cache = !@strstr` (or is it `@!strstr`).

Or maybe merge onto `$glob` the element `'cookie_domain' => $cookie_domain`.

**Comment by bhsmither** [comment]
Also:
```
'standard_url' => preg_replace(array('#^https#i','#/setup&#'),array('http',''), CC_STORE_URL),
```
I believe the ampersand in `#/setup&#` is not correct. Maybe you mean a dollar sign?

**Comment by abrookbanks** [comment]
The first part of your report was fixed with commit dcffd5b72d3807fa35bca8e444a17cfebaba9334

**Comment by abrookbanks** [comment]
```&``` was a mistake! Fixed now. Thanks.
